doc: known-issues: add known issue for clock_gettime() API (#7481)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

Other issues
============

.. rst-class:: v1-9-1 v1-9-0

> The time returned by date_time library is incorrect after one week of uptime
>   The time returned by date_time library is incorrect after one week has elapsed since the system booted because of an issue with clock_gettime() API.

Reviewer: ```suggestion
The time returned by :ref:`lib_date_time` library becomes incorrect after one week of uptime
  The time returned by :ref:`lib_date_time` library becomes incorrect after one week elapses. This is because the system boots due to an issue with clock_gettime() API.
```

Author: The system does not boot due to the issue. I tried to say that the issue becomes visible after device has been running continuously for one week. I think the first sentence is fine and the second sentence could be something like "This is because of an issue with clock_gettime() API."

Author: Updated as discussed offline.

Zephyr
******

.. rst-class:: v1-9-1 v1-9-0

The time returned by clock_gettime() API is incorrect after one week of uptime

>   The time returned by POSIX clock_gettime() API is incorrect after one week has elapsed since the system booted because of an overflow in the uptime conversion.

Reviewer: ```suggestion
  The time returned by POSIX clock_gettime() API becomes incorrect after one week elapses. This is because the system boots due to an overflow in the uptime conversion.
```

Author: Updated as discussed offline.
